src/components/search-bar/story-header.dev.inc.tsx

```diff
@@ -24,6 +27,26 @@ const StoryHeader: React.FC<StoryHeaderProps> = ({ search, userProfile }) => {
   const [isEnabledAdvancedSearch, setIsEnabledAdvancedSearch] = useState(true);
   const headerMenuRef = React.useRef<HTMLDivElement>(null);
 
+  const googleTranslateElementInit = () => {
+    new window.google.translate.TranslateElement(
+      {
+        pageLanguage: "da",
+        autoDisplay: false
+      },
+      "google_translate_element"
+    );
+  };
+
+  useEffect(() => {
+    const addScript = document.createElement("script");
```

Comment:
Before inserting the script with google do the checking if the require api already presented in window. In case it's not do the script insertion.